src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -44,12 +48,38 @@ class EmojiPickerMenu extends Component {
         // If more emojis are ever added to emojis.js this will need to be updated or things will break
         this.unfilteredHeaderIndices = [0, 33, 59, 87, 98, 120, 147];
 
+        // Toggles which keys the search input will listen to
+        // NOTE: these need to be instance members so we can
+        // reference the same event handlers in memory when removing them
+        this.keyToDefaultPreventer = {};
+        ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp'].forEach((key) => {
+            this.keyToDefaultPreventer[key] = (keyBoardEvent) => {
+                if (keyBoardEvent.key === key) {
+                    keyBoardEvent.preventDefault();
+                }
+            };
+        });
+        this.toggleKeysOnSearchInput = (keysToIgnore = [], keysToAccept = []) => {
```

Comment:
I just figured it would be clearer for `toggleKeysOnSearchInput` to go next to `keyToDefaultPreventer` since they're directly related. We have to construct `keyToDefaultPreventer` inside of the constructor.